src/main.cpp

```diff
@@ -6951,13 +6995,31 @@ static int getInitCbhSafeDepth()
     return Params().CbhSafeDepth();
 }
 
+static int getInitScCoinsMaturity()
+{
+    if ( (Params().NetworkIDString() == "regtest") || (Params().NetworkIDString() == "testnet") )
```

Comment:
Why do we need to allow change coins maturity parameter via rpc in a testnet?
Seems, that regtest is enough.

Reply:
You are right, it should probably not be allowed, or it is useless anyway.